custom/auth/ante/fee.go

```diff
@@ -190,23 +225,25 @@ func DeductFees(bankKeeper types.BankKeeper, ctx sdk.Context, acc types.AccountI
 // unit of gas is fixed and set by each validator, can the tx priority is computed from the gas price.
 // Transaction with only oracle messages will skip gas fee check and will have the most priority.
 // It also checks enough fee for treasury tax
-func (fd FeeDecorator) checkTxFee(ctx sdk.Context, tx sdk.Tx, taxes sdk.Coins) (int64, error) {
+func (fd FeeDecorator) checkTxFee(ctx sdk.Context, tx sdk.Tx, taxes sdk.Coins, nonTaxableTaxes sdk.Coins) (int64, bool, bool, error) {
 	feeTx, ok := tx.(sdk.FeeTx)
 	if !ok {
-		return 0, errorsmod.Wrap(sdkerrors.ErrTxDecode, "Tx must be a FeeTx")
+		return 0, false, false, errorsmod.Wrap(sdkerrors.ErrTxDecode, "Tx must be a FeeTx")
 	}
 
 	feeCoins := feeTx.GetFee()
 	gas := feeTx.GetGas()
 	msgs := feeTx.GetMsgs()
 	isOracleTx := isOracleTx(msgs)
+	minGasPrices := fd.taxKeeper.GetEffectiveGasPrices(ctx)
```

Comment:
this can be move into 
```go
if !isOracleTx {
    minGasPrices := fd.taxKeeper.GetEffectiveGasPrices(ctx)

		requiredGasFees := sdk.Coins{}
		if !minGasPrices.IsZero() {}
```